src/cuenta_usuario/templates/registro.html

```diff
@@ -0,0 +1,18 @@
+{% load bootstrap5 %}
+{% bootstrap_css %}
+{% bootstrap_javascript %}
+{% bootstrap_messages %}
+{% block title %}{% endblock %}
+
+{% block content %}
+<h2>Registrar</h2>
+    <div class="card">
+        <div class="card-body">
+            <form role="form" method="post">
+                {% csrf_token %}
+                {% bootstrap_form form %}
+                {% buttons submit='OK' reset="Reset" %}{% endbuttons %}
+            </form>
+        </div>
+    </div>
+{% endblock %}
```

Comment:
```suggestion
{% endblock %}

```

Agrega la línea vacía al final